shared/team-building/index.tsx

```diff
@@ -513,34 +513,41 @@ class TeamBuilding extends React.PureComponent<Props> {
 
     return (
       <>
-        <Kb.List
-          reAnimated={true}
-          items={this.props.searchResults || []}
-          onScroll={this.onScroll}
-          selectedIndex={this.props.highlightedIndex || 0}
-          style={styles.list}
-          contentContainerStyle={styles.listContentContainer}
-          keyboardShouldPersistTaps="handled"
-          keyProperty="key"
-          onEndReached={this._onEndReached}
-          onEndReachedThreshold={0.1}
-          renderItem={(index, result) => (
-            <ResultRow
-              resultForService={this.props.selectedService}
-              username={result.username}
-              prettyName={result.prettyName}
-              displayLabel={result.displayLabel}
-              services={result.services}
-              inTeam={result.inTeam}
-              isPreExistingTeamMember={result.isPreExistingTeamMember}
-              isYou={result.isYou}
-              followingState={result.followingState}
-              highlight={!Styles.isMobile && index === this.props.highlightedIndex}
-              onAdd={() => this.props.onAdd(result.userId)}
-              onRemove={() => this.props.onRemove(result.userId)}
-            />
-          )}
-        />
+        {this.props.searchResults === undefined || this.props.searchResults?.length ? (
```

Comment:
can you factor this `this.props.searchResults === undefined || this.props.searchResults?.length ?` out into its own variable, e.g. `searchResultsNonemptyOrLoading` or maybe `haveEmptySearchResults` and swap the order of the two options? I find it a bit hard to read as is.